src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Drive-by change or related to the PR?

Reply:
It's related in order to see the changed log messages.